### PR TITLE
add x-bows woody keyboard

### PR DESCRIPTION
**Pull request by moyi4681**
add x-bows woody keyboard

### CODE REVIEW

File: keyboards/xbows/woody/info.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+["","","","","","","","","","","","","",{w:2},"",""],
```
Comment (fauxpark) [inline keyboards/xbows/woody/info.json]: This file is in the wrong format.

Please use https://qmk.fm/converter/ to turn it into a proper info.json.

File: keyboards/xbows/woody/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		    KC_LSFT,       KC_Z,     KC_X,     KC_C,  KC_V,  KC_B,  KC_N,    KC_M,  KC_COMM, KC_DOT,   KC_SLSH,  KC_RSFT,           KC_UP,     KC_END,\
+		    KC_LCTL,       KC_LGUI,  KC_LALT,                KC_SPC,                KC_RALT, KC_RCTL,            KC_LEFT,           KC_DOWN,   KC_RIGHT),
+		[1] = LAYOUT( /* FN */
+		    KC_GESC,       KC_F1,    KC_F2,   KC_F3,  KC_F4,   KC_F5,   KC_F6,             KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,   KC_F12,  KC_DEL,   KC_HOME,\
+		    KC_TRNS,       RGB_TOG,  RGB_MOD, RGB_HUI,RGB_HUD, RGB_SAI, RGB_SAD,           RGB_VAI, RGB_VAD, KC_TRNS, KC_PSCR,  KC_SLCK,  KC_PAUS, RESET,    KC_PGUP,\
+	            KC_CAPS,       RGB_SPI,  RGB_SPD, KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,           EEP_RST,  KC_PGDN,\
+		    KC_TRNS,       KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS, KC_TRNS, MAGIC_TOGGLE_NKRO, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,           KC_VOLU,  KC_MUTE,\
+		    KC_TRNS,       KC_TRNS,  KC_TRNS,                  KC_TRNS,                                      KC_TRNS, KC_TRNS,  KC_MPRV,           KC_VOLD,  KC_MNXT),
```
Comment (fauxpark) [inline keyboards/xbows/woody/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/xbows/woody/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+void rgb_matrix_indicators_user(void)
+{
+	    if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/xbows/woody/keymaps/default/keymap.c]:
```suggestion
	    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```

File: keyboards/xbows/woody/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+void matrix_scan_user(void)
+{
+  //user matrix
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record)
+{
+	  return true;
+}
```
Comment (fauxpark) [inline keyboards/xbows/woody/keymaps/default/keymap.c]: These functions do nothing, and can be removed.

File: keyboards/xbows/woody/rules.mk [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+#     clock generation for the USB and other AVR subsections. Do NOT tack on a 'UL'
+#     at the end, this will be done automatically to create a 32-bit value in your
+#     source code.
+#
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
```
Comment (fauxpark) [inline keyboards/xbows/woody/rules.mk]: All of this can be deleted, QMK provides these as defaults now.

File: keyboards/xbows/woody/rules.mk [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (fauxpark) [inline keyboards/xbows/woody/rules.mk]: If you know the bootloader for this keyboard, please replace this block with:

```make
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = atmel-dfu
```

File: keyboards/xbows/woody/readme.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# WOO-DY
+![WOO-DY](https://github.com/moyi4681/qmk_firmware/blob/master/keyboards/xbows/woody/img1.png)
+
+Woo-dy, A 67 Keys Wooden Mechanical Keyboard.
+At once minimalist and stylish, Woo-dy brings a natural calm and relaxation to your working space. The natural wood base made from real walnut or cherry reduces weight, feels great and adds subtle, natural style to any room.
+
+Keyboard Maintainer: X-BOWS
```
Comment (fauxpark) [inline keyboards/xbows/woody/readme.md]: X-BOWS does not make this keyboard. The Kickstarter for it is here: https://www.kickstarter.com/projects/woo-dy/woo-dy-a-67-keys-mechanical-keyboard-that-is-breathing/comments

File: keyboards/xbows/woody/readme.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# WOO-DY
+![WOO-DY](https://github.com/moyi4681/qmk_firmware/blob/master/keyboards/xbows/woody/img1.png)
```
Comment (fauxpark) [inline keyboards/xbows/woody/readme.md]: Please host this image on imgur.com, and delete the one you've committed here.

**Comment by moyi4681** [comment]
@fauxpark  pls check, thanks

File: keyboards/xbows/woody/readme.md (hunk 7 above)
Comment (drashna) [reply keyboards/xbows/woody/readme.md]: I think that this actually needs to be resolved, or the links/wording updated.  

I don't see this keyboard for sale on their site, so listing it as an X-BOWS board seems misleading, if not outright wrong.

Comment (moyi4681) [reply keyboards/xbows/woody/readme.md]: I'm sure it's a new verson of X-BOWS pcb. don't worry about that.:)

Comment (fauxpark) [reply keyboards/xbows/woody/readme.md]: It does seem to be in the carousel on the X-BOWS site now. I don't think it was before. Odd.

File: keyboards/xbows/woody/info.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "",
```
Comment (fauxpark) [inline keyboards/xbows/woody/info.json]:
```suggestion
    "keyboard_name": "WOO-DY", 
```

**Pull request closed without merging**